### DISCUSSION TITLE
Розподіл обов'язків

### DISCUSSION THREAD

**Discussion by GT-740** [ESEI-ZNU/Calculator#4] [General]
Мiй варiант розподiлу:
Розробник №1: **М'якутiков Борис**; Розробник №2: **Кудінов Костянтин**; Розробник №3: **Переверзов Марк**; Розробник №4: **Шустер Кирило**

**Comment by Kisvey** [comment]
Згоден

**Comment by metholyn** [comment]
Згоден
